spec/index.md

```diff
@@ -335,163 +325,131 @@ Each dimension is represented by a key in the `dimensions` object. The object ha
 }
 ```
 
-## Examples
-
-{% assign sorted_pages = site.pages | sort:"order" %}
-{% for page in sorted_pages %}
-  {% if page.category == 'example' %}
-  * [{{ page.title }}]({{ page.url | remove: 'index.html' }}) 
-  {% endif %}
-{% endfor %}
-
-# Content
```

Comment:
Have we lost the content heading entirely. I think the division into structure (form) and content is useful -- though very happy to have comments to the contrary 😄 

It is true that some of the introductory text here could move to a tutorial or a primer ...